media-video/obs-studio/obs-studio-22.0.2.ebuild

```diff
@@ -45,17 +46,24 @@ COMMON_DEPEND="
 	fdk? ( media-libs/fdk-aac:= )
 	imagemagick? ( media-gfx/imagemagick:= )
 	jack? ( virtual/jack )
+	luajit? ( dev-lang/luajit:2 )
+	nvenc? ( media-video/ffmpeg:=[nvenc] )
 	pulseaudio? ( media-sound/pulseaudio )
+	python? ( ${PYTHON_DEPS} )
+	speexdsp? ( media-libs/speexdsp )
```

Comment:
Please switch to `USE=speex` here, which is used elsewhere in tree already.

Reply:
Yes please. speexdsp and speex are two parts of the same software, and while they are independently packaged for some time now, there is no reason to expect we should follow through with more use flag granularity as well.